lib/Transforms/Instrumentation/FatBufferBoundsCheck.cc

```diff
@@ -75,6 +75,10 @@ static cl::opt<bool> UseFatVac("horn-bnd-chk-vac",
                                cl::desc("Use vacuity check for bounds checks"),
                                cl::init(true));
 
+static cl::opt<bool> addIsAllocCheck("add-is-alloc-check",
```

Comment:
option should start with `horn-` or some other prefix that is passed through the cli scripts.

I would use `horn-` because of the other options in this file. We can normalize option prefixes later on.

Reply:
Fixed